packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -68,25 +60,18 @@ function ExportMenu(props: Props) {
 
   return (
     <>
-      <button
+      <Btn
         ref={refs.setReference}
         id={referenceId}
-        className={toolbarStyles.btn}
-        type="button"
+        label={`Export${isSlice ? ' slice' : ''}`}
+        icon={FiDownload}
+        withArrow
         disabled={availableEntries.length === 0}
         aria-haspopup="menu"
-        aria-expanded={isOpen || undefined}
+        aria-expanded={isOpen}
```

Comment:
This is the small ARIA bug: when the popup is closed, the `button` should have attribute `aria-expanded="false"`.